include/library/spdm_lib_config.h

```diff
@@ -270,12 +275,17 @@
 
 
 /* Required scratch buffer size for libspdm internal usage.
- * It maybe used to hold the encrypted/decrypted message and/or last sent/received message.
+ * It may be used to hold the encrypted/decrypted message and/or last sent/received message.
+ * It may be used to hold the large request/response in case of chunking.
+ *
+ * If chunking is not supported, it may be just LIBSPDM_SENDER_RECEIVE_BUFFER_SIZE.
+ * If chunking is supported, it should be at least LIBSPDM_SENDER_RECEIVE_BUFFER_SIZE
+ * LIBSPDM_MAX_SPDM_MSG_SIZE.
  *
  * The value is NOT configurable.
- * The value MAY be chnaged in different libspdm version.
+ * The value MAY be changed in different libspdm version.
  * It is exposed here, just in case the libspdm consumer wants to configure the setting at build time.
  */
-#define LIBSPDM_SCRATCH_BUFFER_SIZE (LIBSPDM_SENDER_RECEIVE_BUFFER_SIZE)
+#define LIBSPDM_SCRATCH_BUFFER_SIZE (LIBSPDM_SENDER_RECEIVE_BUFFER_SIZE + LIBSPDM_MAX_SPDM_MSG_SIZE)
```

Comment:
do we need include LIBSPDM_MAX_SPDM_MSG_SIZE if LIBSPDM_ENABLE_CHUNK_CAP is 0 ?

Reply:
You are right. LIBSPDM_SCRATCH_BUFFER should depend on if LIBSPDM_ENABLE_CHUNK_CAP is enabled or not.